spec/unit/lib/itamae/backend_spec.rb

```diff
@@ -52,5 +52,43 @@ def initialize(_)
         end
       end
     end
+
+    describe Ssh do
+
```

Comment:
actually this is first time for me to write rspec code, so it does not meet your requirement.
if so, rewrite it for the quality!